cmake/config/arm-linux.cmake

```diff
@@ -15,7 +15,7 @@
 include(CMakeForceCompiler)
 
 set(CMAKE_SYSTEM_NAME Linux)
-set(CMAKE_SYSTEM_PROCESSOR arm)
+set(CMAKE_SYSTEM_PROCESSOR armv7l-hf)
```

Comment:
Sorry for the late hit, I didn't realize before that this file also needs a copyright year update.